src/main/assembly/dist/cfg/logback-service.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration scan="true" scanPeriod="1 minute">
-    <appender name="FILE"
+    <appender name="FILE-DOWNLOAD"
```

Comment:
Can't we just leave this one as `name="FILE"`?

Reply:
allright